app/src/Command/LDAPImportCommand.php

```diff
@@ -91,6 +91,14 @@ private function importUsers(): void
 
             foreach ($results as $entry) {
                 $emailAdress = $entry->getAttribute($mailAttribute) ? $entry->getAttribute($mailAttribute)[0] : null;
+                // Verify and recover the email domain
+                $emailDomain = $this->ldapService->extractDomainFromEmail($emailAdress);
+            $domainEntity = $emailDomain ? $this->ldapService->getDomainByName($emailDomain) : null;
```

Comment:
Attention à l’indentation. Aussi, on peut nommer la variable `$domain` : comme on manipule un objet "métier", c’est légèrement plus naturel à la lecture.


```suggestion
                $domain = $emailDomain ? $this->ldapService->getDomainByName($emailDomain) : null;
```